receiver/mongodbreceiver/metadata.yaml

```diff
@@ -37,6 +37,24 @@ attributes:
     enum:
       - hit
       - miss
+  lock_type:
+    description: The Type of Lock
+    enum:
+      - parallel_batch_write_mode
+      - replication_state_transition
+      - global
+      - database
+      - collection
+      - mutex
+      - metadata
+      - oplog
+  lock_mode:
+    description: The mode of Lock
```

Comment:
This does not communicate anything more than the attribute name. Please provide a more detailed description.

Reply:
Updated description with more details